src/System.Net.Http/tests/FunctionalTests/DefaultCredentialsTest.cs

```diff
@@ -153,5 +198,50 @@ private class CredentialWrapper : ICredentials
             public NetworkCredential GetCredential(Uri uri, String authType) => 
                 InnerCredentials?.GetCredential(uri, authType);
         }
+
+        private class AuthenticatedProxy : IWebProxy
+        {
+            ICredentials _credentials;
+            Uri _proxyUri;
+
+            public AuthenticatedProxy(ICredentials credentials)
+            {
+                _credentials = credentials;
+
+                string host = Environment.GetEnvironmentVariable("HTTP_INTERNAL_PROXYSERVER");
```

Comment:
Originally #2383 was going to solve this long-term. But it seems that using environment variables is a good cross-plat, easy to use, solution.

I'll consolidate those settings into one of the Common test files (i.e. HttpTestServers.cs) for now.